src/coreclr/nativeaot/Runtime/wasm/FinalizerHelpers.SingleThreaded.cpp

```diff
@@ -42,7 +42,7 @@ void RhEnableFinalization()
     // here as it will deadlock the GC.
 }
 
-EXTERN_C NATIVEAOT_API void __cdecl RhWaitForPendingFinalizers(UInt32_BOOL allowReentrantWait)
+EXTERN_C void __cdecl RhWaitForPendingFinalizers(UInt32_BOOL allowReentrantWait)
```

Comment:
```suggestion
EXTERN_C void QCALLTYPE RhWaitForPendingFinalizers(UInt32_BOOL allowReentrantWait)
```
Can sync this with the other definition while we're here.